### ISSUE TITLE
Swipe scroll (Drag) stopped working on touchscreen after 1 swipe on card

### ISSUE THREAD

**Issue by ivictbor** [kutlugsahin/vue-smooth-dnd#134]
Here is a video with steps to reproduce on demo page: https://screencast-o-matic.com/watch/cqliQsULpr

**Comment by prileso** [comment]
The class "smooth-dnd-disable-touch-action" stays on the body preventing scrolling. I'm guessing it should be removed once the touch ends but for some reason isn't. Really loving the drag/drop effects with this library so will look into it some more.

**Comment by ivictbor** [comment]
@prileso yes, I also love vue-smooth-dnd, we've been using it on big live project per a year already, but still living with these minor bugs, though it is great work anyway, really hope @kutlugsahin will return to the project at least to review and fix most important things someday, or maybe some of us will find a time to create a pull request

**Comment by ful-stackz** [comment]
Hello! I had to face this same issue, and managed to overcome it with a pretty clean hack. Here is my solution:

```javascript
<template>
  <div @touchend="fixActionRestriction">
    <draggable-container>
      <draggable />
    </draggable-container>
  </div>
</template>

<script>
import { Container as DraggableContainer, Draggable } from "vue-smooth-dnd";

export default {
  components: {
    DraggableContainer,
    Draggable
  },
  methods: {
    fixActionRestriction() {
      document.body.classList.remove(
        "smooth-dnd-no-user-select",
        "smooth-dnd-disable-touch-action"
      );
    }
  }
}
</script>
```

**Comment by prileso** [comment]
@ful-stackz Awesome, seems to do the trick! I also noticed that a fix for this bug has been made in the smooth-dnd project and it seems people have made their own forks(?) for it. I'm not git-savy enough to know how to utilize those though:
https://github.com/kutlugsahin/smooth-dnd/pull/67

**Comment by FrozenHearth** [comment]
> Hello! I had to face this same issue, and managed to overcome it with a pretty clean hack. Here is my solution:
> 
> ```js
> <template>
>   <div @touchend="fixActionRestriction">
>     <draggable-container>
>       <draggable />
>     </draggable-container>
>   </div>
> </template>
> 
> <script>
> import { Container as DraggableContainer, Draggable } from "vue-smooth-dnd";
> 
> export default {
>   components: {
>     DraggableContainer,
>     Draggable
>   },
>   methods: {
>     fixActionRestriction() {
>       document.body.classList.remove(
>         "smooth-dnd-no-user-select",
>         "smooth-dnd-disable-touch-action"
>       );
>     }
>   }
> }
> </script>
> ```

This works, but I get this error;

`Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.`

**Comment by ful-stackz** [comment]
@FrozenHearth Hey there, I'm glad it works for you! The error on the other hand is from the `vue-smooth-dnd` library itself (or `smooth-dnd` I'm not sure). Checkout #133, which describes the error and is completely unrelated to the hack I discovered.

I am also experiencing the same errors, which haven't caused any issues, yet. It seems to be that the browser just doesn't like you intercepting scroll events.

**Comment by FrozenHearth** [comment]
@ful-stackz Thanks for the hack though! Really appreciated. Is there any way to suppress this error? I've searched on Stackoverflow and tried some fixes, but they're not working.

**Comment by ful-stackz** [comment]
@FrozenHearth No, I'm afraid you can't supress errors raised from the browser 😢 They are supposed to be taken seriously, hence the inability to disable them. Which in this case is annoying. On the bright side, though, mobile users don't have an (easy) access to a console, so who cares 😅

**Comment by ffxsam** [comment]
Should this be a PR? IMO this is a major bug that ought to be officially patched.

**Comment by tschombes** [comment]
> Hello! I had to face this same issue, and managed to overcome it with a pretty clean hack. Here is my solution:
> 
> ```js
> <template>
>   <div @touchend="fixActionRestriction">
>     <draggable-container>
>       <draggable />
>     </draggable-container>
>   </div>
> </template>
> 
> <script>
> import { Container as DraggableContainer, Draggable } from "vue-smooth-dnd";
> 
> export default {
>   components: {
>     DraggableContainer,
>     Draggable
>   },
>   methods: {
>     fixActionRestriction() {
>       document.body.classList.remove(
>         "smooth-dnd-no-user-select",
>         "smooth-dnd-disable-touch-action"
>       );
>     }
>   }
> }
> </script>
> ```

Could anyone explain please how to implement this fix? This bug is currently a Showstopper for our NC Setup. Would be great If anyone could Help, thx

**Comment by ivictbor** [comment]
> 

Just take a cerefull look at code above. What it does is declares method which removes 2 classes from body. This method is called on @touchend="fixActionRestriction" which should be attached to div which wraps draggeble container. Works great for our product, you can check it on https://tracklify.com It is canban board, uses vue-snooth-dnd you can login with Google
